src/analyses/varEq.ml

```diff
@@ -151,6 +152,7 @@ struct
     let bt =  unrollTypeDeep (Cilfacade.typeOf b) in
     type_may_change_t a bt
 
+  (* TODO: why unused? how different from below? *)
```

Comment:
They are very similar, but the code below handles more cases, but is also considerably more weird.